services/composite-keys-manager.js

```diff
@@ -8,6 +8,10 @@ function CompositeKeysManager(model, schema, record) {
     var where = {};
     var primaryKeyValues = recordId.split(GLUE);
 
+    primaryKeyValues.forEach(function (key, index) {
```

Comment:
Can you add a comment to explain why we have to do that here (something that could happen with a bad model configuration)?